build/binder_typemaps/csharp.json

```diff
@@ -1,5 +1,5 @@
 {
-  "GLenum": "int",
+  "GLenum": "GLEnum",
```

Comment:
Actually, what happens if you remove this line altogether? We do extended mapping for enums (i.e. for NV enums, ARB enums, etc)

Reply:
Seems like parameters are fine, but return types are broken. For example, in IArbDirectStateAccess.gen.cs:

`GLenum CheckNamedFramebufferStatus([Flow(FlowDirection.In)] uint framebuffer, [Flow(FlowDirection.In)] GLEnum target);`

Reply:
Awesome, glad to know that that's been hooked up to some extent. Should be an easy fix, go into TypeMapper.cs (in the converter) and change the MapEnums function to this:
![image](https://user-images.githubusercontent.com/11160611/61638050-864d6d00-ac90-11e9-9c51-ef312d6dffac.png)
You'll need to do a fullrun

Reply:
Done, now it works without that line in the typemapper.

Reply:
After giving Rider some more time to process everything, a bunch of errors popped up as a result. For example, in INVPathRendering.gen.cs:

`unsafe void MatrixLoad([Flow(FlowDirection.In)] EXT mode, [Count(Count = 16), Flow(FlowDirection.In)] float* m);`

EXT doesn't exist in this project.

Reply:
Hmm will have a look at it myself.